README.md

```diff
@@ -76,3 +76,12 @@ or_like_this = [
 
 There are plenty of other examples (ordinary and extraordinary) in the
 [test_file](https://github.com/DSpeckhals/python-indent/blob/master/spec/test_file.py).
+
+### This isn't working for me!
+
+Sorry to hear that! Hopefully one of the following will help you fix the problem.
+
+* Check is that you are using "soft tabs" (i.e. spaces). Hard tabs (i.e. `'\t'`) are incompatible with PEP8 compliant indenting so this package gives up if the editor is using hard tabs.
```

Comment:
Technically, I think tabs are acceptable, just not for "continued indentation." I would re-word it surrounding that point.